.changeset/rare-waves-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-common': patch
+---
+
+Add `CatalogEntityPermission` convenience type.
```

Comment:
Perhaps mention that it's `@alpha` and point to `/alpha`?

Reply:
:+1: done in 669dec17eb.